package.json

```diff
@@ -1,27 +1,63 @@
 {
-  "name": "pixi-spine",
-  "version": "1.0.1",
+  "name": "pixi-spinejs",
+  "version": "1.0.0",
   "description": "Spine implementation for pixi v3",
   "author": "Mat Groves",
   "contributors": [
     "Chad Engler <chad@pantherdev.com>",
     "Richard Davey <rdavey@gmail.com>"
   ],
-  "main": "index.js",
+  "main": "./src/index.js",
   "homepage": "https://github.com/pixijs/pixi-spine",
-  "bugs": "https://github.com/pixijs/pixi-spine/issues",
+  "bugs": {
+    "url": "https://github.com/pixijs/pixi-spine/issues"
+  },
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/pixijs/pixi-spine.git"
   },
   "keywords": [
-    "pixi",
-    "spine",
-    "2d",
-    "render"
+  	"pixi",
+  	"spine",
+  	"2d",
+  	"render"
   ],
-  "peerDependencies": {
-    "pixi.js": "^3.0.0-rc4"
+  "peerDependencies":{
+  	"pixi.js": "^3.0.0-rc4"
+  },
+  "scripts": {
+    "test": "gulp test",
+    "build": "gulp build",
+    "docs": "./node_modules/.bin/jsdoc -c ./gulp/util/jsdoc.conf.json"
+  },
+  "devDependencies": {
+    "browserify": "^9.0.3",
+    "chai": "^2.1.1",
+    "del": "^1.1.0",
+    "gulp": "^3.8.10",
+    "gulp-cached": "^1.0.2",
+    "gulp-jshint": "^1.9.0",
+    "gulp-mocha": "^2.0.0",
+    "gulp-plumber": "^0.6.6",
+    "gulp-rename": "^1.2.0",
+    "gulp-uglify": "^1.0.2",
+    "gulp-util": "^3.0.1",
+    "ink-docstrap": "^0.5.2",
+    "jsdoc": "^3.3.0-alpha13",
+    "jshint-summary": "^0.4.0",
+    "require-dir": "^0.1.0",
+    "run-sequence": "^1.0.2",
+    "vinyl-buffer": "^1.0.0",
+    "vinyl-source-stream": "^1.0.0",
+    "watchify": "^2.2.1"
+  },
+  "dependencies": {
+  	"brfs": "~1.4.0"
```

Comment:
updated
I left it in there because it was an option for how to configure stuff.